### PR TITLE
1.7.x updates

### PR DESCRIPTION
**Pull request by shamusx**
1.7 updates
- Leverage tsb to manage istio intermediate certificate (istiod cacerts for all CPs)
- Toggle some environment variables
- enable isolation boundaries

### CODE REVIEW

File: prep_managementplane_values.sh [hunk 1]
```diff
@@ -29,6 +29,9 @@ $(awk '{printf "        %s\n", $0}' < xcp-central-cert.key)
     rootca: |
 $(awk '{printf "      %s\n", $0}' < ca.crt)
 spec:
+  certIssuer:
```
Comment (smarunich) [inline prep_managementplane_values.sh:32]: if we do this should we get rid off... of 

https://github.com/tetrateio/tsb-helm-install-demos/blob/9017b41044270842a81b3c142a9521d49fab6058/prep_controlplane_values.sh#L33 and L36? what about elastic any chance there too?

File: certs-gen/certs-gen.sh [hunk 2]
```diff
@@ -129,10 +129,7 @@ extendedKeyUsage = serverAuth, clientAuth
 subjectAltName   = @alt_names
 # you will have to use the following DNS name and URI because it will be checked by the TSB
 [ alt_names ]
-DNS.1 = xcp.tetrate.io
-URI.1 = spiffe://xcp.tetrate.io/central
 DNS.2 = ${TSB_FQDN}
-DNS.3 = ${TSB_FQDN}:9443
 EOF
```
Comment (smarunich) [inline certs-gen/certs-gen.sh:134]: Technically we can call certs-gen.sh - generate rootCA or can we actually get rid of it completely as in theory tsb provided certs are self-signed anyway? so technically we don't need the script anymore?

Comment (shamusx) [reply certs-gen/certs-gen.sh:134]: I left the generate to allow us to use FQDN in certificate, as it looks cert-management currently will create`certificate` based on status of service and doesn't have a way for us to provide SAN property.

The other piece is we need to get CA certs to controlplane as these wont get passed, while we can use what TSE does:
`tctl apply -f "$FOLDER/cluster-$CLUSTER_NAME.yaml" -o yaml | yq '.spec.installTemplate.helm' > "$FOLDER/$CLUSTER_NAME-controlplane_values.yaml"`
But the problem with this is we impose the `yq` tool where for this repo we were aiming to use built in tooling.

WDYT @smarunich

File: prep_managementplane_values.sh (hunk 1 above)
Comment (shamusx) [reply prep_managementplane_values.sh:32]: these don't get auto populated - we still need to provide CP the CA's.

**Review by smarunich — APPROVED** [review]

**Pull request merged**